### PR TITLE
Added 25 new quotes

### PR DESCRIPTION
**Pull request by lucas-fine**
### Checklist 
- [x] I have read the [`CODE_OF_CONDUCT.md`](https://github.com/Miodec/monkeytype/blob/master/CODE_OF_CONDUCT.md) and the [`CONTRIBUTING.md`](https://github.com/Miodec/monkeytype/blob/master/CONTRIBUTING.md)
- [x] I checked if my PR has any bugs or other issues that could reduce the stability of the project
- [x] I understand that the maintainer has the right to reject my contribution and it may not get accepted.

Resolves #

### CODE REVIEW

File: static/quotes/english.json [hunk 1]
```diff
@@ -32902,6 +32902,162 @@
...
+      "text": "For a brief stint in high school, Scout joined the track team in one of his many schemes to pick up girls. He was kicked off the team after three when everyone realized he was 23-years-old and also not enrolled in school.",
+      "source": "Team Fortress 2",
+      "length": 221,
+      "id": 5542
+    },
+    {
+      "text": "Join your crewmates in a multiplayer game of teamwork and betrayal! Play online or over local wifi with 4-10 players as you attempt to hold your spaceship together and return back to civilization. But beware...as there may be an alien impostor aboard!",
+      "source": "Amoung Us, Inner Sloth",
+      "length": 251,
+      "id": 5543
```
Comment (Miodec) [inline static/quotes/english.json:32928]: I dont think this quote is suitable.

File: static/quotes/english.json [hunk 2]
```diff
@@ -32902,6 +32902,162 @@
...
+    },
+    {
+      "text": "The Industrial Revolution and its consequences have been a disaster for the human race. They have greatly increased the life-expectancy of those of us who live in “advanced” countries, but they have destabilized society, have made life unfulfilling, have subjected human beings to indignities, have led to widespread psychological suffering (in the Third World to physical suffering as well) and have inflicted severe damage on the natural world. The continued development of technology will worsen the situation. It will certainly subject human beings to greater indignities and inflict greater damage on the natural world, it will probably lead to greater social disruption and psychological suffering, and it may lead to increased physical suffering even in “advanced” countries.",
```
Comment (Miodec) [inline static/quotes/english.json:32931]: `“advanced”` replace the quotes here with `\"`

File: static/quotes/english.json [hunk 3]
```diff
@@ -32902,6 +32902,162 @@
...
+      "text": "Combat automaton which became self-aware. Ancient technology allows manipulation of Orbs.",
+      "source": "Slay The Spire",
+      "length": 82,
+      "id": 5552
+    },
+    {
+      "text": "A deadly huntress from the foglands. Eradicates foes with daggers and poisons.",
+      "source": "Slay The Spire",
+      "length": 78,
+      "id": 5553
```
Comment (Miodec) [inline static/quotes/english.json:32988]: Im not a fan of these either, sorry.

File: static/quotes/english.json [hunk 4]
```diff
@@ -32902,6 +32902,162 @@
...
+    },
+    {
+      "text": "You happen upon a group of what looks like purple fire spirits dancing around a large bonfire. The spirits toss small bones and fragments into the fire, which brilliantly erupts each time. As you approach, the spirits all turn to you, expectantly...",
+      "source": "Slay The Spire",
+      "length": 249,f
```
Comment (Miodec) [inline static/quotes/english.json:32993]: remove the `f`

File: static/quotes/english.json [hunk 5]
```diff
@@ -32902,6 +32902,162 @@
...
+      "text": "You don't remember me? I'll give you a clue... Does April the 3rd mean anything to you? I believe that was the day of our first encounter. You look like you might be remembering something...",
+      "source": "Hotline miami",
+      "length": 190,
+      "id": 5556
+    },
+    {
+      "text": "It's the most innovative shooter I've played in years.",
+      "source": "SUPERHOT",
+      "length": 54,
+      "id": 5557
```
Comment (Miodec) [inline static/quotes/english.json:33012]: Please remove

File: static/quotes/english.json [hunk 6]
```diff
@@ -32902,6 +32902,162 @@
...
+      "id": 5561
+    },
+    {
+      "text": "There's something very strange about this place... the layout seems to change all the time. I've never seen the same room twice!",
+      "source": "Splunky",
+      "length": 128,
+      "id": 5562
+    },
+    {
+      "text": "Greetings! My name is Yang, and this is my journal. dusty maps and rumors of amazing treasures have led me here, deep under the desert.",
```
Comment (Miodec) [inline static/quotes/english.json:33045]: Capitalise `dusty`

File: static/quotes/english.json [hunk 7]
```diff
@@ -32902,6 +32902,162 @@
...
+      "source": "Splunky",
+      "length": 135,
+      "id": 5563
+    },
+    {
+      "text": "Washington, Washington, six foot eight, weighs a f***ing ton, opponents beware, opponents beware, he's coming, he's coming, he's coming",
```
Comment (Miodec) [inline static/quotes/english.json:33051]: Remove the censorship

File: static/quotes/english.json [hunk 8]
```diff
@@ -32902,6 +32902,162 @@
...
+      "source": "Washington, TheRealBradNeely",
+      "length": 135,
+      "id": 5564
+    },
+    {
+      "text": "Do you want a banana? (Do you want a banana?) Peel it down and go, \"Mm-mmm mm-mmm.\" Do you want a banana? (Do you want a banana?) Dis banana for you",
+      "source": "Banana Man, Tally Hall",
+      "length": 50,
+      "id": 5565
+    },
```
Comment (Miodec) [inline static/quotes/english.json:33061]: Remove please.

File: static/quotes/english.json [hunk 9]
```diff
@@ -32902,6 +32902,162 @@
...
+      "text": "Throughout the galaxy a legend is told, on a distant planet a grim fortress stood, until a deadly force parted the heavens and descended upon the keep. Though brought to ruin, the ashes of that place hold an artifact of impossible power - a gun that can kill the past. Over time the fortress was rebuilt, and some who hear the legend would risk everything for another shot - to claim their prize and make what was done, undone they must - Enter The Gungeon.",
+      "source": "Enter The Gungeon",
+      "length": 457,
+      "id": 5548
+    },
+    {
+      "text": "I report to God. You report to me",
+      "source": "Terry A. Devis",
+      "length": 33,
+      "id": 5549
```
Comment (Miodec) [inline static/quotes/english.json:32964]: Too short i think

File: static/quotes/english.json [hunk 10]
```diff
@@ -32902,6 +32902,162 @@
...
+      "text": "You happen upon a group of what looks like purple fire spirits dancing around a large bonfire. The spirits toss small bones and fragments into the fire, which brilliantly erupts each time. As you approach, the spirits all turn to you, expectantly...",
+      "source": "Slay The Spire",
+      "length": 249,f
+      "id": 5554
+    },
+    {
+      "text": "Greetings, neighbor! The name's Crazy Dave. But you can just call me Crazy Dave. Listen, I've got a surprise for you. But first, I'll need you to clear your lawn. Use your shovel and dig up those plants! LET THE DIGGING COMMENCE!",
+      "source": "Plants Vs Zombies",
+      "length": 229,
+      "id": 5555
```
Comment (Miodec) [inline static/quotes/english.json:33000]: Please remove

File: static/quotes/english.json [hunk 11]
```diff
@@ -32902,6 +32902,162 @@
...
+      "text": "Hi am Heavy Weapons Guy, and this is my weapon. She weighs one hundred fifty kilograms and fires two hundred dollar, custom-tooled cartridges at ten thousand rounds per minute. It costs four hundred thousand dollars to fire this weapon, for twelve seconds.",
+      "source": "Team Fortess 2, Meet the Heavy",
+      "length": 256,
+      "id": 5546
+    },
+    {
+      "text": "Schizophrenia spectrum and other psychotic disorders include schizophrenia, other psychotic disorders, and schizotypal (personality) disorder. They are defined by abnormalities in one or more of the following five domains: delusions, hallucinations, disorganized thinking (speech), grossly disorganized or abnormal motor behavior (including catatonia), and negative symptoms.",
+      "source": "DSM V",
+      "length": 375,
+      "id": 5547
```
Comment (Miodec) [inline static/quotes/english.json:32952]: Remove please

**Review by Miodec — CHANGES_REQUESTED** [review]
Marked all the mistakes I found as well as quotes that I don't think are good enough. Please recheck again for proper grammar as well, there might be something I missed.

**Comment by lucas-fine** [comment]
Made requested updates

**Review by Miodec — APPROVED** [review]

**Pull request merged**